com.unity.visualeffectgraph/Editor/GraphView/Views/VFXPaste.cs

```diff
@@ -80,9 +80,15 @@ static bool CanPasteNode(Node node, string openedAssetPath)
 
         public static bool CanPaste(VFXView view, object data)
         {
+            var content = data?.ToString();
+            if (string.IsNullOrEmpty(content))
+            {
+                return false;
+            }
+            
             try
             {
-                var serializableGraph = JsonUtility.FromJson<SerializableGraph>(data.ToString());
+                var serializableGraph = JsonUtility.FromJson<SerializableGraph>(content);
```

Comment:
The formatting is failing on this file 🤔
You can run it manually using:
```
perl $HOME\unity-meta\Tools\Format\format.pl --nobackups --hgroot $(pwd) .
```

Reply:
resolve at https://github.com/Unity-Technologies/Graphics/pull/6864/commits/a012ffe1d5208836a58fd819354c688f3c49ef00